app/react-components/Gene.js

```diff
@@ -179,6 +179,11 @@ let Gene = (() => {
         return {
           ...x,
           num_affected_cases_all: x.occurrence.length,
+          num_affected_cases_by_project: x.occurrence.reduce((acc, o) => ({
+            ...acc,
+            [o.case.project.project_id]: acc[o.case.project.project_id] ? acc[o.case.project.project_id] + 1 : 1
```

Comment:
I'm not saying do this but does this work?

`[o.case.project.project_id]: (acc[o.case.project.project_id] || 0) + 1`